scss/main.scss

```diff
@@ -1141,6 +1141,151 @@ p.cardText {
   align-items: center;
 }
 
+.content-container {
+  margin-left: auto;
+  margin-right: auto;
+  padding: 80px 96px;
+  max-width: 1440px;
+  height: max-content;
+  .content-header {
+    h2 {
+      padding: 16px 0px 16px 0px;
+      margin-bottom: 0px;
+      margin-left: auto;
+      margin-right: auto;
+      width: 600px;
+      height: auto;
+      font-family: $font-family-base;
+      font-style: normal;
+      font-weight: bold;
+      font-size: 3rem;
+      line-height: 62px;
+      color: $darker-grey;
+    }
+    p {
+      margin-bottom: 24px;
+      margin-left: auto;
+      margin-right: auto;
+      width: 600px;
+      height: auto;
+      font-family: $font-family-base;
+      font-style: normal;
+      font-weight: normal;
+      font-size: 1.25rem;
+      line-height: 30px;
+      color: $darker-grey;
+    }
+  }
+  
+  @media (max-width: 768px) {
+    padding: 64px 48px;
+    .content-header {
+      h2 {
+        width: 672px;
+      }
+      p {
+        width: 672px;
+      }
+    }
+  }
+
+  @media (max-width: 651px) {
+    .content-header{
+      h2 {
+        font-size: 2rem;
+        width: 555px;
+      }
+      p {
+        font-size: 1.125;
+        line-height: 23px;
+        width: 555px;
+      }
+    }
+  }
+
+  @media (max-width: 360px) {
+    padding: 48px 24px;
+    .content-header {
+      h2 {
+        font-size: 2rem;
+        width: 312px;
+      }
+      p {
+        font-size: 1.125rem;
+        line-height: 23px;
+        width: 312px;
+      }
+    }
+  }
+ }
+  
+ .quote-content {
+  position: relative;
+  max-width: 1440px;
+  .quote-paragraph-container {
+    padding-top: 80px;
+    padding-left: 107px;
+    height: max-content;
+    transition: ease all .5s;
```

Comment:
No need for these transition properties. Can we remove all of them, please? (Animating things like width and padding are very expensive).